src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Mef/ExportProviderCache.cs

```diff
@@ -284,7 +284,9 @@ public Assembly LoadAssembly(string assemblyFullName, string codeBasePath)
                 var assemblyName = new AssemblyName(assemblyFullName);
                 if (!string.IsNullOrEmpty(codeBasePath))
                 {
+#pragma warning disable SYSLIB0044 // Type or member is obsolete
```

Comment:
📝 The same change was made in Roslyn for this test helper that we borrowed from them.